glue/qt/glue_application.py

```diff
@@ -518,6 +524,27 @@ def _create_actions(self):
         a.triggered.connect(nonpartial(self._choose_save_session))
         self._actions['session_save'] = a
 
+        from glue.config import importers
+        if len(importers) > 0:
```

Comment:
@astroforg I think that if you don't define a custom importer, then the "load dataset" menu item disappears from the file menubar (at least that's what happens for me).